addon/components/cfb-form-editor/question.js

```diff
@@ -196,6 +230,14 @@ export default Component.extend(ComponentQueryManager, {
 
   submit: task(function*(changeset) {
     try {
+      if (!this.get("slug")) {
+        const namespace = this.options.get("namespace");
+        if (namespace !== undefined) {
```

Comment:
`if (namespace)` should be enough

Reply:
It should. Except when it's "0" or something else that happens to be falsy when converted. As long as I don't know what `namespace` could be, I'd rather not use a truthy condition.

Reply:
I'd rather impose some conditions on the values that can be used as namespace. It should always be a string for one, which solves the issue with "0". It can't be an empty string, otherwise it'll be treated as if no namespace is present (which makes total sense). Also, are you currently making sure that the concatenated result (namespace + actual slug) is slugified together?

Reply:
Namespace is now a computed property and an empty string if not present.